src/Sql/Ddl/Column/Column.php

```diff
@@ -1,110 +1,79 @@
- * @license   http://framework.zend.com/license/new-bsd New BSD License
+ * @see       https://github.com/zendframework/zend-db for the canonical source repository
+ * @copyright Copyright (c) 2005-2019 Zend Technologies USA Inc. (https://www.zend.com)
+ * @license   https://github.com/zendframework/zend-db/blob/master/LICENSE.md New BSD License
  */
 
+declare(strict_types=1);
+
 namespace Zend\Db\Sql\Ddl\Column;
 
 use Zend\Db\Sql\Ddl\Constraint\ConstraintInterface;
 
 class Column implements ColumnInterface
 {
-    /**
-     * @var null|string|int
-     */
+    /** @var null|string|int */
     protected $default;
 
-    /**
-     * @var bool
-     */
+    /** @var bool */
     protected $isNullable = false;
 
-    /**
-     * @var string
-     */
+    /** @var string */
     protected $name = '';
 
-    /**
-     * @var array
-     */
+    /** @var array */
     protected $options = [];
 
-    /**
-     * @var ConstraintInterface[]
-     */
+    /** @var ConstraintInterface[] */
     protected $constraints = [];
 
-    /**
-     * @var string
-     */
+    /** @var string */
     protected $specification = '%s %s';
 
-    /**
-     * @var string
-     */
+    /** @var string */
     protected $type = 'INTEGER';
 
-    /**
-     * @param null|string $name
-     * @param bool        $nullable
-     * @param mixed|null  $default
-     * @param mixed[]     $options
-     */
-    public function __construct($name = null, $nullable = false, $default = null, array $options = [])
+    public function __construct(string $name = '', bool $nullable = false, ?$default = null, array $options = [])
     {
         $this->setName($name);
         $this->setNullable($nullable);
         $this->setDefault($default);
         $this->setOptions($options);
     }
 
-    /**
-     * @param  string $name
-     * @return self Provides a fluent interface
-     */
-    public function setName($name)
+    public function setName(string $name) : self
     {
-        $this->name = (string) $name;
+        $this->name = $name;
+
         return $this;
     }
 
-    /**
-     * @return null|string
-     */
-    public function getName()
+    public function getName() : string
     {
         return $this->name;
     }
 
-    /**
-     * @param  bool $nullable
-     * @return self Provides a fluent interface
-     */
-    public function setNullable($nullable)
+    public function setNullable(bool $nullable) : self
     {
-        $this->isNullable = (bool) $nullable;
+        $this->isNullable = $nullable;
+
         return $this;
     }
 
-    /**
-     * @return bool
-     */
-    public function isNullable()
+    public function isNullable() : bool
     {
         return $this->isNullable;
     }
 
     /**
-     * @param  null|string|int $default
-     * @return self Provides a fluent interface
+     * @param null|string|int $default
+     * @return self
      */
-    public function setDefault($default)
+    public function setDefault(?$default) : self
```

Comment:
also here, question mark, please check all places

Reply:
changed back, but open for discussion based on the PSR-12 draft